### PR TITLE
[WIP] React Native framework

### PR DESCRIPTION
**Pull request by dangeross**
This PR adds `sdk-react-native` framework to the libs available in breez-sdk. Following the initial steps of #45 this is a manually implemented framework, which builds the bindings available in `sdk-bindings` and copies the resulting libraries into the android and ios projects.  Currently only `mnemonicToSeed` is implemented from the SDK as proof-of-concept and included in the framework is an example project that can be built to test this.

### CODE REVIEW

File: libs/sdk-react-native/android/src/main/java/com/breezsdk/BreezSDKModule.kt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+class BreezSDKModule(reactContext: ReactApplicationContext) : ReactContextBaseJavaModule(reactContext) {  
+    override fun getName(): String {
+        return "BreezSDK";
+    }
+
+    @ReactMethod
+    fun mnemonicToSeed(mnemonic: String, promise: Promise) {
+        try {
+            var seed = mnemonicToSeed(mnemonic);
+            var response = writableMapOf("type" to "seed", "data" to seed);
```
Comment (roeierez) [inline libs/sdk-react-native/android/src/main/java/com/breezsdk/BreezSDKModule.kt:19]: What is the advantage of adding the "type" key here? What is the advantage of this structure vs returning ReadableArray as a single value?

File: libs/sdk-react-native/android/src/main/java/com/breezsdk/MapResponse.kt [hunk 2]
```diff
@@ -0,0 +1,60 @@
+package com.breezsdk
+
+import com.facebook.react.bridge.Arguments
+import com.facebook.react.bridge.WritableArray
+import com.facebook.react.bridge.WritableMap
+
+fun writableMapOf(vararg values: Pair<String, *>): WritableMap {
```
Comment (roeierez) [inline libs/sdk-react-native/android/src/main/java/com/breezsdk/MapResponse.kt:7]: Isn't it better to use the readonly version for the return types (ReadableMap)?

**Review by roeierez — COMMENTED** [review]
@dangeross this looks really good! I left some comments to understand better your decisions.
Thank you very much!

File: libs/sdk-react-native/android/src/main/java/com/breezsdk/MapResponse.kt (hunk 2 above)
Comment (dangeross) [reply libs/sdk-react-native/android/src/main/java/com/breezsdk/MapResponse.kt:7]: You're right, these return types should be ReadableArray or ReadableMap.

File: libs/sdk-react-native/android/src/main/java/com/breezsdk/BreezSDKModule.kt (hunk 1 above)
Comment (dangeross) [reply libs/sdk-react-native/android/src/main/java/com/breezsdk/BreezSDKModule.kt:19]: I intend to change this to just map the SDK responses as much as possible.

**Review by roeierez — APPROVED** [review]
@dangeross LGTM!

**Comment by roeierez** [comment]
@dangeross added you as a collaborator, feel free to merge.

**Pull request merged**